meta-genivi-dev/recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -112,10 +112,9 @@ SRC_URI[index.sha256sum] = "86114b93f1f51aaf0aec3af0751d214b351f4ff9839ba031315c
 
 SYSTEMD_SERVICE_${PN} = "sota_client.service"
 
-DEPENDS += " openssl "
+DEPENDS += " openssl dbus "
```

Comment:
oh, i missed that fact somehow. :( I'll use the "first day after vacation" excuse again. :) @chbae thanks for the quick action.